exatomic/va/vroa_funcs.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015-2018, Exa Analytics Development Team
+# Distributed under the terms of the Apache License 2.0
+"""
+VROA functions
+##################
+Functions for VROA calculations
+"""
+from numba import jit, prange
+import numpy as np
+
+@jit(nopython=True)
+def _backscat(C_au, beta_g, beta_A):
+    return 4.* (24 * beta_g + 8 * beta_A)
+
+@jit(nopython=True)
+def _forwscat(C_au, alpha_g, beta_g, beta_A):
+    return 4.* (180 * alpha_g + 4 * beta_g - 4 * beta_A)
```

Comment:
From the quick read in the Numba docs it seems like it would. I only knew about jit so that's why I used. I'll edit the function with `@vectorize` decorator